django/db/models/fields/json.py

```diff
@@ -0,0 +1,93 @@
+import json
+
+from django.core import exceptions
+from django.utils.translation import gettext_lazy as _
+
+from . import Field
+
+
+def convert_jsonfield_value(value, expression, connection):
+    if connection.vendor == 'oracle':
+        return value.read()
+    return value
+
+
+class JSONField(Field):
+    description = _('A JSON object')
+    default_error_messages = {
+        'invalid': _("Value must be valid JSON."),
+    }
+    _default_hint = ('dict', '{}')
+
+    def __init__(self, encoder=None, decoder=None, default=dict, *args, **kwargs):
+        self.encoder, self.decoder = encoder, decoder
+        super().__init__(*args, **kwargs)
+
+    def db_check(self, connection):
+        data = self.db_type_parameters(connection)
+        if connection.vendor in ('mysql', 'sqlite'):
+            return 'JSON_VALID(%(qn_column)s)' % data
+        if connection.vendor == 'oracle':
```

Comment:
use `elif`

Reply:
Didn't know "unnecessary `else` after `return`" rule doesn't apply to `elif`s. Thanks!